components/RoadmapPage/RoadmapPage.tsx

```diff
@@ -0,0 +1,218 @@
+import Swiper, { EffectCreative, Mousewheel } from "swiper"
+import { useMediaQuery } from "usehooks-ts"
+import Slider from "../../shared/Slider"
+import Stage from "./Stage"
+import Layout from "../Layout"
+import { StageData } from "./types"
+
+Swiper.use([Mousewheel])
+
+const RoadmapPage = () => {
+  const stages: StageData[] = [
+    {
+      backImg: "/assets/Roadmap/mission.svg",
+      label: "Mission & Vision",
+      date: "2023-01-31",
+    },
+    {
+      backImg: "/assets/Roadmap/curate.svg",
+      label: "Curate collective",
+      date: "2023-02-28",
+    },
+    {
+      backImg: "/assets/Roadmap/pendants.svg",
+      label: "Airdrop pendants",
+      date: "2023-03-31",
+    },
+    {
+      backImg: "/assets/Roadmap/build.svg",
```

Comment:
optional: would be nice to split this out into it's own`.json` file to not clutter up the HTML.

love this architecture you've built. 🏗️ 

very easy to modify / update over time. ✏️